.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
+
```

Comment:
There are 2 'introductory to accessibility' http://digitalgov.lndo.site/admin/content?title=&type=resources&status=All